funnel/models/profile_membership.py

```diff
@@ -0,0 +1,169 @@
+# -*- coding: utf-8 -*-
+
+from sqlalchemy.ext.associationproxy import association_proxy
+
+from coaster.sqlalchemy import immutable, with_roles
+
+from . import db
+from .membership import ImmutableMembershipMixin
+from .profile import Profile
+from .user import User
+
+__all__ = ['ProfileAdminMembership']
+
+
+class ProfileAdminMembership(ImmutableMembershipMixin, db.Model):
+    """
+    A users can be an administrator of a profile and optionally an owner.
+    Owners can manage other administrators.
+    """
+
+    __tablename__ = 'profile_admin_membership'
+
+    # List of role columns in this model
+    __data_columns__ = ('is_owner',)
+    __parent_column__ = 'profile_id'
```

Comment:
Replace with `parent_id = db.synonym('profile_id')` below, after defining the `profile_id` column.